File: sdks/thunkmetrc-go/wrapper/models/patient_check_in.go
package models

type PatientCheckIn struct {
    CheckInDate string `json:"CheckInDate,omitempty"`
    CheckInLocationId int64 `json:"CheckInLocationId,omitempty"`
    CheckInLocationName string `json:"CheckInLocationName,omitempty"`
    Id int64 `json:"Id,omitempty"`
    PatientLicenseNumber string `json:"PatientLicenseNumber,omitempty"`
    RegistrationExpiryDate string `json:"RegistrationExpiryDate,omitempty"`
    RegistrationStartDate string `json:"RegistrationStartDate,omitempty"`
}

File: sdks/thunkmetrc-cs/ThunkMetrc.Wrapper/Models/PatientCheckIn.cs
using System;
using System.Collections.Generic;

namespace ThunkMetrc.Wrapper.Models
{
    public class PatientCheckIn
    {
        [global::System.Text.Json.Serialization.JsonPropertyName("CheckInDate")]
        public required string CheckInDate { get; set; }
        [global::System.Text.Json.Serialization.JsonPropertyName("CheckInLocationId")]
        public required int CheckInLocationId { get; set; }
        [global::System.Text.Json.Serialization.JsonPropertyName("CheckInLocationName")]
        public required string CheckInLocationName { get; set; }
        [global::System.Text.Json.Serialization.JsonPropertyName("Id")]
        public required int Id { get; set; }
        [global::System.Text.Json.Serialization.JsonPropertyName("PatientLicenseNumber")]
        public required string PatientLicenseNumber { get; set; }
        [global::System.Text.Json.Serialization.JsonPropertyName("RegistrationExpiryDate")]
        public required string RegistrationExpiryDate { get; set; }
        [global::System.Text.Json.Serialization.JsonPropertyName("RegistrationStartDate")]
        public required string RegistrationStartDate { get; set; }
    }
}
